configs/nextflow.hopper.config

```diff
@@ -172,7 +183,7 @@ profiles {
     params.umi = true
     params.gatkcnv = false
     params.distributed = false
-    params.delly = true
+    params.delly = true         // delly is no longer supported
```

Comment:
Left this one in as a reminder that the profile relied on delly, as all processes / code related to the latter are removed with this update.

Reply:
Hmm. Should we remove the full profile? Seems strange to remove parts of it while still allowing executing the dysfunctional profile